compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -5377,14 +5377,25 @@ OMR::Z::CodeGenerator::replaceInst(TR::Instruction* old, TR::Instruction* curr)
       }
    }
 
+// TODO: This should really be a common code generator API. Push this up to the common code generator.
```

Comment:
Thanks.  Can you reference this issue in the comment in the code?  Otherwise it may be difficult to find the exact issue that was created.